test/cmd/help.sh

```diff
@@ -6,23 +6,21 @@ os::test::junit::declare_suite_start "cmd/help"
 # This test validates the help commands and output text
 
 # verify some default commands
-os::cmd::expect_success 'openshift'
 os::cmd::expect_success 'kubectl'
 os::cmd::expect_success 'oc'
 os::cmd::expect_success 'oc ex'
-os::cmd::expect_success 'origin'
 
 # help for root commands must be consistent
-os::cmd::expect_success_and_text 'openshift' 'Application Platform'
+os::cmd::expect_failure_and_text 'openshift' 'Application Platform'
```

Comment:
I'd rather not fix it this way.  Why does it die?

Reply:
[NewCmdOpenShift makes use of the DefaultSubCommandRun helper](https://github.com/openshift/origin/blob/master/pkg/cmd/openshift/openshift.go#L94)

https://github.com/kubernetes/kubernetes/pull/50672 updated that helper to exit with code 1